source/includes/steps-saving-favorite-queries.yaml

```diff
@@ -0,0 +1,38 @@
+title: Open query history
+level: 4
+ref: saving-favorite-queries
+content: |
+  Click the :guilabel:`History` button at the top of the pipeline 
+  builder.
+
+  .. figure:: /images/compass/query-history-view.png
+     :figwidth: 720 px
+     :alt: Click history button
+---
+title: Select favorites
+level: 4
+ref: click-favorites-button
+content: |
+  Click the :guilabel:`Star` button.
+
+  .. figure:: /images/compass/query-add-favorite.png
+     :figwidth: 316 px
+     :alt: Click star button
+---
+title: Name your query
+level: 4
+ref: name-your-query
+content: |
+  Enter a name for your query.
+
+  .. figure:: /images/compass/query-add-favorite-name.png
+     :figwidth: 316 px
+     :alt: Save favorite query
+---
+title: Save
+level: 4
+ref: save-your-query
+content: |
+  Click :guilabel:`Save`.
```

Comment:
For something like this we can omit the `content` and keep the action in the title. We can avoid the redundancy.